Comments/Migrations/20190709103350_SeedingQuestionTypeYesNo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Comments.Migrations
+{
+    public partial class SeedingQuestionTypeYesNo : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
```

Comment:
do we need the migration to InsertData _and_ the HasData in the context? i'd have thought one way or t'other (and preferably the latter).

Reply:
@AndyChildNice would you be able to take a look at this?